runtime/server_http_request.go

```diff
@@ -651,6 +651,20 @@ func (req *ServerHTTPRequest) UnmarshalBody(
 	return true
 }
 
+// WriteBytesBody replace the request body
+// we only change things cached in this layer, not in the raw httpRequest
+// Assumption is that encoding does not change
+func (req *ServerHTTPRequest) WriteBytesBody(
+	body []byte) bool {
+	// Replace the cached body bytes and fix dependent header
+	req.rawBody = body
+	if _, ok := req.Header.Get("Content-Length"); ok {
+		req.Header.Set("Content-Length", strconv.Itoa(len(body)))
+	}
+
+	return true
```

Comment:
Seems like it always return `true`? Why do we need return a bool?

Reply:
Bool was because other similar functions were doing so.  And my 1st implementation had an error return with false.  But looking at this, it makes no sense to return anything.  Will change.